config/initializers/new_framework_defaults_5_2.rb

```diff
@@ -8,7 +8,7 @@
 
 # Make Active Record use stable #cache_key alongside new #cache_version method.
 # This is needed for recyclable cache keys.
-# Rails.application.config.active_record.cache_versioning = true
+Rails.application.config.active_record.cache_versioning = true
```

Comment:
@bladealslayer 
cache_versioning work with "dalli" gem. "readthis" gem never noticed cache_versioning sooner does not work with cache_versioning. Please check out what is written in the variables for this https://github.com/sharetribe/sharetribe/blob/rails-5.2.3/config/environments/production.rb#L72

There is 3rd party gem "memcachier" involved here. I don't know it is working or not.